lib/misc.js

```diff
@@ -89,4 +103,17 @@ module.exports = (api) => {
     return ee;
   };
 
+  api.common.restLeft = (
+    // Transforms function with (args, arg1 .. argN, callback) arguments
```

Comment:
@tshemsedinov, You've described `toAsync` contract but as you say [here](https://github.com/metarhia/metasync/issues/145#issuecomment-309774531), `restLeft` is more generalized.